govc/library/update.go

```diff
@@ -86,7 +86,7 @@ func (cmd *update) Run(ctx context.Context, f *flag.FlagSet) error {
 		item := &library.Item{
 			ID:          t.ID,
 			Name:        cmd.name,
-			Description: cmd.desc,
+			Description: &cmd.desc,
```

Comment:
Looks like with this change, using `govc library.update -n $name` will erase the description if `-d $description` is not specified.  I think for now, we could maintain the current behavior by doing something like so:
```diff
modified   govc/library/update.go
@@ -84,10 +84,13 @@ func (cmd *update) Run(ctx context.Context, f *flag.FlagSet) error {
 		return m.UpdateLibrary(ctx, &t)
 	case library.Item:
 		item := &library.Item{
-			ID:          t.ID,
-			Name:        cmd.name,
-			Description: cmd.desc,
+			ID:   t.ID,
+			Name: cmd.name,
 		}
+		if cmd.desc != "" {
+			item.Description = &cmd.desc
+		}
+
 		t.Patch(item)
 		return m.UpdateLibraryItem(ctx, item)
 	default:
```

Reply:
Hey Doug, thanks for the review. I have changed library description from string to a pointer in the latest revision. That revision also updated govc, to make it be able to update the library and item description to an empty string, so behavior can be consistent with the API.